pkg/fixtures/triggers.go

```diff
@@ -69,18 +69,31 @@ var Events = map[string]string{
 }
 
 // BuildFromFixture creates a new fixture struct for a file
-func BuildFromFixture(fs afero.Fs, apiKey, stripeAccount, apiBaseURL, jsonFile string) (*Fixture, error) {
+func BuildFromFixture(fs afero.Fs, apiKey string, stripeAccount string, skip []string, overrides []string, additions []string, removals []string, apiBaseURL string, jsonFile string) (*Fixture, error) {
```

Comment:
when multiple params have the same type, the type can be given only once. In that case that might help make the code a bit cleaner:

```go
func BuildFromFixture(fs afero.Fs, apiKey, stripeAccount string, skip, overrides, additions, removals []string, apiBaseURL, jsonFile string) (*Fixture, error) {
```

Additionally if we reorder some params (appending the new customization params at the end, we save a bit more clarity):
```go
func BuildFromFixture(fs afero.Fs, apiKey, stripeAccount, apiBaseURL, jsonFile string, skip, overrides, additions, removals []string) (*Fixture, error) {
```

Just a nit and personal pref :)